src/tasks/copy-certs.ts

```diff
@@ -0,0 +1,78 @@
+import { V1Secret, V1SecretList } from '@kubernetes/client-node'
+import { env } from '../common/envalid'
+import { getApiClient } from '../common/k8s'
+import { getFilename, OtomiDebugger, terminal } from '../common/utils'
+
+const cmdName = getFilename(__filename)
+const debug: OtomiDebugger = terminal(cmdName)
+
+let targetNamespace = 'istio-system'
```

Comment:
Looks good! I just am a bit confused why you didn't pass `targetNamespace` as a parameter to `copyTeamTlsSecrets`, because I had to ctrl+f this variable to find out what it's used for and it also introduces a side effect ("A side effect is when a function relies on, or modifies, something outside its parameters to do something")

Reply:
Did you see it is prefixed with `let`? It is dependent on `hasCloudLB`. So yes, there is an intended side effect.

Reply:
I thought you wanted pure functions.